ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -732,6 +734,38 @@ export default class AdvancedTab extends PureComponent {
     );
   }
 
+  renderUse4ByteResolutionToggle() {
+    const { t } = this.context;
+    const { use4ByteResolution, setUse4ByteResolution } = this.props;
+    return (
+      <div
+        ref={this.settingsRefs[12]}
```

Comment:
Should this be in advanced or security settings?

Reply:
I'm pretty sure we mentioned it being in both the advanced and security settings tabs, unless I'm misunderstanding this @pedronfigueiredo @kevinghim : 
>We can start by adding a toggle to the settings UI and linking it to the preferences controller. This toggle should be visible in the advanced privacy settings section as well as in the advanced privacy settings section during user onboarding.

Reply:
@vthomas13 kevin is mentioning two different places - Onboarding and the settings screen. Though "Advanced" and "Privacy" are two independent locations in the settings screen @kevinghim  can you clarify? Do we have a precedent for a setting being in two different screens in settings?

Reply:
removed in b4f0a7e861edb7ef20b183b296c932928e3d3db3